forecastSrc/bbc_forecast_data.py

```diff
@@ -1,144 +1,167 @@
                 except ValueError:
-                    maxTemp     = int(descInfo[0].split(':')[1].lstrip()[0:1])
+                    maxTemp = int(descInfo[0].split(":")[1].lstrip()[0:1])
 
-        minIndex = 1+maxIndex
+        minIndex = 1 + maxIndex
         try:
-            minTemp     = int(descInfo[minIndex].split(':')[1].lstrip()[0:3])
+            minTemp = int(descInfo[minIndex].split(":")[1].lstrip()[0:3])
         except ValueError:
             try:
-                minTemp     = int(descInfo[minIndex].split(':')[1].lstrip()[0:2])
+                minTemp = int(descInfo[minIndex].split(":")[1].lstrip()[0:2])
             except ValueError:
-                minTemp     = int(descInfo[minIndex].split(':')[1].lstrip()[0:1])
+                minTemp = int(descInfo[minIndex].split(":")[1].lstrip()[0:1])
 
-        #maxTemp = str(maxTemp)
-        #minTemp = str(minTemp)
+        # maxTemp = str(maxTemp)
+        # minTemp = str(minTemp)
 
-        windDir     = descInfo[2+maxIndex].split(':')[1].lstrip()
-        windSpeed   = "{:.1f}".format(int(descInfo[3+maxIndex].split(':')[1].lstrip().replace('mph','')) * 0.44704)
-        pressure    = descInfo[5+maxIndex].split(':')[1].lstrip().replace('mb','')
-        humidity    = descInfo[6+maxIndex].split(':')[1].lstrip().replace('%','')
+        windDir = descInfo[2 + maxIndex].split(":")[1].lstrip()
+        windSpeed = "{:.1f}".format(
+            int(descInfo[3 + maxIndex].split(":")[1].lstrip().replace("mph", ""))
+            * 0.44704
+        )
+        pressure = descInfo[5 + maxIndex].split(":")[1].lstrip().replace("mb", "")
+        humidity = descInfo[6 + maxIndex].split(":")[1].lstrip().replace("%", "")
 
         dayGiven = titleInfo[0][0:3]
         if dayGiven == startDay:
             date = dates[0]
-        elif dayGiven == daysOfWeek[(daysOfWeek.index(startDay) + 1)%7]:
+        elif dayGiven == daysOfWeek[(daysOfWeek.index(startDay) + 1) % 7]:
             date = dates[1]
         else:
             date = dates[2]
-        raw_forecasts.append({
-            'date'          : date,
-            #'from'          : '',
-            #'to'            : '',
-            'symbol'        : symbol,
-            'precipitation' : '',
-            'wind_dir'      : windDir,
-            'wind_speed'    : windSpeed,
-            #'temperature'   : '',
-            'temp_min'      : minTemp,
-            'temp_max'      : maxTemp,
-            'pressure'      : pressure,
-            'humidity'      : humidity
-        })
-        if date in dated_forecast.keys():
-            dated_forecast[date].append({
+        raw_forecasts.append(
+            {
+                "date": date,
                 #'from'          : '',
                 #'to'            : '',
-                'symbol'        : symbol,
-                'precipitation' : '',
-                'wind_dir'      : windDir,
-                'wind_speed'    : windSpeed,
+                "symbol": symbol,
```

Comment:
here again I see that the alignment of the code has been disturbed. Previously all the lines were written such that ":" was aligned in one column. This makes for better code-readability in my opinion. Could you please revert?